app/routers/profile.py

```
@@ -0,0 +1,338 @@
+    JobStatusResponse,
+    ProfileMetricsResponse
+)
+
+router = APIRouter(prefix="/api/v1/profiles", tags=["User Profiles"])
+
+
+@router.get("/{user_id}", response_model=UserProfileResponse)
+async def get_user_profile(
+    user_id: int = Path(..., gt=0, description="사용자 ID (양수)"),
+    include_metadata: bool = Query(False, description="메타데이터 포함 여부"),
+    session: AsyncSession = Depends(get_async_session)
```

Ruff (0.11.9)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)